tests/test_qnode.py

```diff
@@ -153,33 +153,33 @@ def qf(x):
         @qml.qnode(self.dev2)
         def qf(x):
             qml.RX(x, wires=[0])
-            return qml.expval.PauliZ(wires=0), 0.3
+            return qml.expval(qml.PauliZ(wires=0)), 0.3
```

Comment:
We can return constant values from QNodes?!

Reply:
a new "feature"?